sycl/test/inline-asm/include/asmhelper.h

```diff
@@ -0,0 +1,44 @@
+#include <CL/sycl.hpp>
+
+#include <iostream>
+#include <vector>
+
+struct WithOneWriteAccessor {
```

Comment:
```suggestion
struct WithOneBuffer {
```

Since it doesn't actually store any accessors